Gopkg.toml

```diff
@@ -55,7 +55,7 @@
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/operatorkit/pull/319 gets merged.